spring-cloud-netflix-core/src/test/java/org/springframework/cloud/netflix/feign/FeignHttpClientUrlTests.java

```diff
@@ -58,7 +58,7 @@
 @RunWith(SpringJUnit4ClassRunner.class)
 @SpringBootTest(classes = FeignHttpClientUrlTests.TestConfig.class, webEnvironment = WebEnvironment.DEFINED_PORT, value = {
 		"spring.application.name=feignclienturltest", "feign.hystrix.enabled=false",
-		"feign.okhttp.enabled=false" })
+		"spring.cloud.httpclientfactories.ok.enabled=false" })
```

Comment:
Yes, they are documented in Spring Cloud Commons.  However, this property doesnt even need to be set so I am going to reverse it :)